src/extensions/LinkBubblePluginView.js

```diff
@@ -132,13 +103,7 @@ class LinkBubblePluginView {
 			return
 		}
 
-		if (this.#updateDebounceTimer) {
-			clearTimeout(this.#updateDebounceTimer)
-		}
-
-		this.#updateDebounceTimer = window.setTimeout(() => {
-			this.updateFromSelection(view)
-		}, updateDelay)
+		debounce(() => this.updateFromSelection.bind(this)(view), updateDelay, { immediate: true })
```

Comment:
Does this debounce actually have an effect? As far as I know it would still call every time, unless you would call the returned function

Reply:
@max-nextcloud to the rescue :grimacing: I still struggle to understand the different scopes of JS functions :see_no_evil:

Reply:
Ah sorry, now I got what you mean. Indeed, debounce just returns a function that needs to be called. I thought it would be about the scope of  `this` in `updateFromSelection()`. Anyways, should be fixed now :relaxed: